src/main/java/com/iri/training/repository/HobbyRepositoryImpl.java

```diff
@@ -34,28 +34,30 @@ public HobbyRepositoryImpl() throws IOException {}
 
 	@Override
 	public Hobby getHobbyById(Long hobbyId) throws SQLException {
-		logger.debug("ENTERED getHobbyById for id " + hobbyId);
+		logger.debug("ENTERED getHobbyById for hobbyId: " + hobbyId);
 
 		final Hobby hobby;
 
 		String sql = property.getString("RETRIEVE_HOBBY");
 		jdbcTemplate = new JdbcTemplate(dataSource);
 		hobby = jdbcTemplate.query(sql, new Object[]{hobbyId}, new HobbyResultSetExtractor());
 
-		logger.debug("EXITING getHobbyById for id " + hobbyId);
+		logger.debug("RETRIEVED Hobby: " + hobby.toString());
+		logger.debug("EXITING getHobbyById for hobbyId: " + hobbyId);
```

Comment:
Delete "RETRIEVED" and only print "EXITING" with the hobby object (it will contain the hobbyId). No need to use toString, as it may throw a null pointer exception if the object was not found in the DB.